RishiSrivastava.json

```
@@ -0,0 +1,27 @@
+{
+	"name": "Rishi Srivastava",
+	"displayStatsPublic": true,
+	"type": "personal",
+	"bio": "Frontend Developer, Coding enthusiast",
+	"links": [{
+			"name": "Follow me on Twitter",
+			"url": "https://twitter.com/sri26_rishi",
+			"icon": "FaTwitter"
+		},
+		{
+			"name": "Connect on Linkedin",
+			"url": "https://www.linkedin.com/in/sri-rishi/",
+			"icon": "FaLinkedin"
+		},
+		{
+			"name": "Follow me on Github",
+			"url": "https://github.com/sri-rishi",
+			"icon": "FaGithub"
+		},
+    {
+			"name": "Follow me on Hashnode",
+			"url": "https://hashnode.com/@sririshi",
+			"icon": "FaHashnode"
```

You probably be using an invalid icon here. It should be `SiHashnode`.

```suggestion
			"icon": "SiHashnode"
```

---

As mentioned I have made changes